asreview/logging/utils.py

```diff
@@ -101,3 +102,31 @@ def loggers_from_dir(data_dir, prefix="result"):
         loggers[log_file] = logger_class(log_fp=log_fp, read_only=True)
 
     return loggers
+
+
+def logger_from_file(data_fp):
```

Comment:
Is this a confusing name? `loggers_from_dir`?

Reply:
The documentation was copy/pasted. I have updated the documentation.